aten/src/ATen/DLConvertor.h

```diff
@@ -11,11 +11,50 @@
 namespace at {
 
 TORCH_API ScalarType toScalarType(const DLDataType& dtype);
-TORCH_API DLManagedTensor* toDLPack(const Tensor& src);
-TORCH_API Tensor fromDLPack(DLManagedTensor* src);
-TORCH_API Tensor
-fromDLPack(DLManagedTensor* src, std::function<void(void*)> deleter);
+TORCH_API DLManagedTensorVersioned* toDLPack(const Tensor& src);
```

Comment:
Is this API used by C++ libraries? This would be a BC-breaking change for these users right?

Reply:
I think it's being used by PyTorch/XLA. Yes, it's definitely BC-breaking. I was thinking that, since DLPack 1.0 should be the new default, the old version should have the name with a suffix. However, now that you brought this up, not being BC-breaking sounds more important.

In summary, I will change the names so that we are not BC-breaking.